src/main/webapp/jsp/studentHome.jsp

```diff
@@ -7,7 +7,7 @@
 <c:set var="jsIncludes">
   <script type="text/javascript" src="/js/studentHome.js"></script>
 </c:set>
-<ts:studentPage pageTitle="TEAMMATES - Student" bodyTitle="Student Home" jsIncludes="${jsIncludes}">
+<ts:studentPage pageTitle="Home" bodyTitle="Student Home" jsIncludes="${jsIncludes}">
```

Comment:
I feel that it's mainly caused by student being a special category where we don't add the suffix. As long as `Home [Admin]` and `Home [Instructor]` are consistent, I feel it's fine.